rcl_interfaces/msg/ParameterDescriptor.msg

```diff
@@ -19,16 +19,16 @@ string additional_constraints
 # If 'true' then the value cannot change after it has been initialized.
 bool read_only false
 
-# If this bounded sequence contains a floating_point_range constraint it must
-# be enforced.
-# FloatingPointRange consist of a from_value a to_value and a step.
-# FloatingPointRange and IntegerRange are mutually exclusive
+
+# If any of the following sequences are not empty, then the constraint inside of
+# them apply to this parameter.
+
+# FloatingPointRange consists of a from_value, a to_value and a step.
```

Comment:
Please include the Oxford comma (it's a subjective choice, not right or wrong, but we do include it elsewhere in our code base):

>  FloatingPointRange consists of a from_value, a to_value, and a step.